lte/gateway/c/core/oai/tasks/amf/amf_cn.cpp

```diff
@@ -72,6 +72,27 @@ static int amf_cn_authentication_res(amf_cn_auth_res_t* const msg) {
   OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
 }
 
+//------------------------------------------------------------------------------
+static int amf_cn_implicit_deregister_ue(const amf_ue_ngap_id_t ue_id) {
+  int rc = RETURNok;
```

Comment:
rc is used only while returning. So we can capture the return value of amf_proc_deregistration_request and return the rc

Reply:
Code has been modified, now `rc` is being used.